packages/core/src/Stores/client-store.js

```diff
@@ -613,10 +618,26 @@ export default class ClientStore extends BaseStore {
         return this.dxtrade_accounts_list.some(account => account.account_type === 'real');
     }
 
+    get has_real_derivez_login() {
+        return this.derivez_accounts_list.some(account => account.account_type === 'real');
+    }
+
     hasAccountErrorInCFDList = (platform, account_type) => {
         if (!this.is_logged_in) return false;
-
-        const list = platform === CFD_PLATFORMS.MT5 ? this.mt5_login_list : this.dxtrade_accounts_list;
+        let list;
+        switch (platform) {
+            case 'mt5':
```

Comment:
in case try not to use strings instead use enums or object so we dont need to update strings everywhere.
```suggestion
            case CFD_PLATFORMS.MT5:
```